### ISSUE TITLE
Can't Clear Module Notifications on iOS app

### ISSUE THREAD

**Issue by yashogunjal** [crux-bphc/CMS-iOS#109]
I use the iOS app updated on 17Jan 2021 and I can't clear the notifications of modules and so for me to clear them I have to individually open each notice/upload under the module to clear those notifications. Please look into it.

**Comment by chaubss** [comment]
Could you please elaborate a bit?
When you open the app notifications should be cleared from your notification center automatically.

**Comment by yashogunjal** [comment]
Like I had already opened all the notifications when I had seen them on the
site. Yet when I installed the app there were 100+ notifications from each
module.

Moreover if I want the notifications to go away, I need to individually
check and open each and every topic inside the module for its notifications
to vanish. It would have been great if there were any button to *mark as
read *all the notifications without opening them.

Thanks and regards,
Yashovardhan Gunjal (2020 batch)
